src/client/pages/Project/Changelog/index.js

```diff
@@ -99,6 +108,7 @@ class Changelog extends Component {
     }
 
     const { params: { id }, data } = this.props
+
```

Comment:
Usually the changelog graph is ordered in descending revision number, since revision dates are not guaranteed to be monotonically increasing over the lifetime of the repository. Sorting by date will not guarantee that we will be able to render a sensible changelog graph (at least it will quite possibly often have disconnected parts).
Unless there's a compelling reason, I'd avoid ordering and just accept the natural ordering.